Signal/src/environment/NotificationsManager.m

```diff
@@ -20,7 +22,8 @@
 @interface NotificationsManager ()
 
 @property SystemSoundID newMessageSound;
-@property (nonatomic, readonly) id<ContactsManagerProtocol> contactsManager;
+@property NSMutableDictionary<NSString *, UILocalNotification *> *currentNotifications;
```

Comment:
done. also added asserts to make sure we're only inspecting and mutating it on main thread.